lld/test/ELF/aarch64-got-relocations-pauth.s

```diff
@@ -78,6 +78,79 @@ _start:
   adrp x1, :got_auth:zed
   add  x1, x1, :got_auth_lo12:zed
 
+#--- ok-tiny.s
+
+# RUN: llvm-mc -filetype=obj -triple=aarch64-none-linux ok-tiny.s -o ok-tiny.o
```

Comment:
`none` is also an OS component and is misleading when used together with `linux`.

Just use `aarch64` (or `aarch64-linux` when it's important to mention Linux, usually it's not)

Reply:
Thanks for suggestion, changed to `aarch64` in 3c80d75c601db4574bd0edd6b89dbd332cab5f2e. Also applied to #113815